packages/code-highlight/tests/unit/ui-code-highlight.spec.ts

```diff
@@ -0,0 +1,242 @@
+import { shallowMount, mount } from '@vue/test-utils'
+import { ref } from 'vue'
+import { SupportedLanguajes, UiCodeHighlight } from '../../src'
+import { prismClass } from '../../src/utils'
+
+describe('ui-code-highlight.vue', () => {
+  it('should render line-numbers class when props.linenumbers is set true', async () => {
+    const wrapper = shallowMount(UiCodeHighlight, {
+      props: {
+        code: `p {
```

Comment:
I suggest this code string could be a constant and you can re-use it in this and the next test case